graph/src/data/subgraph/mod.rs

```diff
@@ -568,12 +566,31 @@ pub struct Mapping {
     pub link: Link,
 }
 
+// TODO: This seems to be a part of the stdlib (FuturesOrdered.collect()), but I can't get it working
```

Comment:
Wanna try again together?

Reply:
I can try again. I just wasn't sure it was worth delaying the launch of the network since this task is taking longer than expected.